2_process/src/write_csv_fcn.R

```diff
@@ -0,0 +1,17 @@
+#' # Define write_csv_fcn() function
+#' 
+#' The `write_csv_fcn()` saves a copy of the processed data in csv format.
```

Comment:
Since you are using some of the `roxygen` syntax, I would suggest giving this a prefix
```suggestion
#' @description The `write_csv_fcn()` saves a copy of the processed data in csv format.
```